testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -40,4 +40,103 @@ TestNodeProxy : UnitTest {
 		this.assertEquals(proxy.isPlaying, false, "Setting the proxy's source should not set isPlaying = true");
 	}
 
+	test_asCode_basic {
+		var codeString = "a = NodeProxy.new(Server.fromName( 'TestNodeProxy' ));\n";
+
+		this.assertEquals(proxy.asCode.postcs, codeString,
+			"asCode-posting basic nodeproxy should post valid source code.",
+			false);
+	}
+
+	test_asCode_single {
+		var codeString = "a = NodeProxy.new(Server.fromName( 'TestNodeProxy' )).source_({ DC.ar });\n";
+		proxy.source = { DC.ar };
+
+		this.assertEquals(proxy.asCode, codeString,
+			"asCode-posting single-source nodeproxy should post simple source form.",
+			false
+		);
+	}
+
+	test_asCode_multi {
+		var asCodeString =
+		"(
+a = NodeProxy.new;
+a[5] = { DC.ar };
+a[10] = { DC.ar(0.01) };
+
+);
+";
```

Comment:
make this consistent with other string literals

Reply:
done